edge-modules/api-proxy-module/build/linux/check-submodules.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
What is this file used for? Does this file check sub-modules under api-proxy folder?  Does api-proxy folder have submodules?

Reply:
This script is used to build the project, why this file named as "check-submodule.sh"?

Reply:
you told me that this file is deleted, please check.

Reply:
Deleted.